bot/airtable.py

```diff
@@ -0,0 +1,85 @@
+from pyairtable import Table
```

Comment:
^ The DS_Store files should be deleted and *.DS_Store should be added to the .gitignore